maxlen.xml

```diff
@@ -97,8 +97,9 @@
                 in ROAs except in some specific cases. The recommendations complement and extend
                 those in RFC 7115. The document also discusses the creation of ROAs for facilitating
                 the use of Distributed Denial of Service (DDoS) mitigation services. Considerations
-                related to ROAs and origin validation in the context of destination-based Remote
-                Triggered Black Hole (RTBH) filtering are also highlighted.
+                related to ROAs and origin validation in the context of destination-based Remotely
+                Triggered Discard Route (RTDR) (elsewhere referred to as "Remotely Triggered
```

Comment:
no need to mention RTBH here, because you also mention this elsewhere

Reply:
I wanted to give readers maximum chance to grok that they're the same, so I added that everywhere the acronym is expanded.
I don't think it hurts.

Reply:
Fine